sep.go

```diff
@@ -1,6 +0,0 @@
-package hlsp
```

Comment:
This file is not needed because we will not write specification extensions. They will be (optionally) passed to the parser.Parse method.

Reply:
I had no idea about go.mod, very good this